docs/hosting-a-server-with-northstar/dedicated-server/hosting-on-windows.md

```diff
@@ -0,0 +1,58 @@
+# Hosting on Windows
+
+## Hardware
+
+{% hint style="info" %}
+TODO: If you have experience in hosting Northstar servers and want to share your knowledge, please open a [pull request](https://github.com/R2Northstar/NorthstarWiki/pulls).
+{% endhint %}
```

Comment:
This doesn't feel fitting for Windows hosting page tbh ^^

Reply:
tooooo much copy paste